cg/meta/upload/gt.py

```diff
@@ -126,3 +127,16 @@ def _is_variant_file(genotype_file: housekeeper_models.File):
 
     def _get_genotype_files(self, version_id: int) -> list:
         return self.hk.files(version=version_id, tags=["genotype"]).all()
+
+    @staticmethod
+    def genotype_check(case_obj: models.Family) -> bool:
```

Comment:
```suggestion
    def is_suitable_for_genotype_upload(case_obj: models.Family) -> bool:
```
Usually with functions returning `booleans` you try to use `is_` in the first part of the name. Also, I think this describes much more what is the intent of the function.